src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,85 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.SequenceGenerator;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+public class User {
```

Comment:
Add the @Entity annotation.
You must also add the annotation
@Table (name = "users") for
to rename the table, since User is a hibernate reserved word.

```suggestion
@Entity
@Table(name="users")
public class User {
```